WooCommerce/Classes/ViewModels/ManualTrackingViewModel.swift

```diff
@@ -128,6 +129,12 @@ final class AddTrackingViewModel: ManualTrackingViewModel {
     }
 
     let providerCellAccessoryType = UITableViewCell.AccessoryType.disclosureIndicator
+    var isEmptyState: Bool {
+        (shipmentProvider == nil && trackingNumber.isNilOrEmpty) ? true : false
```

Comment:
You can add directly`(shipmentProvider == nil && trackingNumber.isNilOrEmpty)`,without the need of adding the ternary operator` ? true : false`

Reply:
Thanks for the tip!